src/DevTools/setupTestWrapper.tsx

```diff
@@ -110,7 +127,32 @@ export const setupTestWrapperTL = <T extends OperationType>({
       return MockPayloadGenerator.generate(operation, mockResolvers)
     })
 
-    return { ...view, env }
+    const mockResolveLastOperation = (mockResolvers: MockResolvers) => {
+      let operation
+
+      act(() => {
+        env.mock.resolveMostRecentOperation(relayOperation => {
+          operation = relayOperation
+          return MockPayloadGenerator.generate(operation, mockResolvers)
+        })
+      })
+
+      // Cast here to get around use-before-assign rule
+      const _operation = operation as OperationDescriptor
```

Comment:
Not sure how else to do this @araujobarret

Reply:
If you look at the [relay v14 docs](https://relay.dev/docs/v14.0.0/guides/testing-relay-components/#relaymockenvironment-api-overview) it says that `mockEnvironment.resolveMostRecentOperation()` is actually a sugar for ` env.mock.resolve(env.mock.getMostRecentOperation())`. So I would suggest just calling these methods proceduratlly:
```ts
const operation = env.mock.getMostRecentOperation()
act(() => {
  env.mock.resolve(operation, MockPayloadGenerator.generate(operation, mockResolvers))
})
return operation, etc

Reply:
This is a nice solution to the Q, updated